snuba/utils/streams/kafka.py

```diff
@@ -180,8 +200,11 @@ def revocation_callback(
                     on_revoke(streams)
             finally:
                 for stream in streams:
+                    if stream in self.__staged_offsets:
+                        del self.__staged_offsets[stream]
```

Comment:
It might make sense to warn if this occurs — this means there were things waiting to commit that weren't committed before revocation.